Sources/TuistCache/Utilities/CacheFrameworkBuilder.swift

```diff
@@ -65,6 +65,7 @@ public final class CacheFrameworkBuilder: CacheArtifactBuilding {
         try await xcodeBuildController.build(
             projectTarget,
             scheme: scheme.name,
+            destination: nil,
```

Comment:
To get this caching functionality to work correctly is it important that I use the `osVersion` and `deviceName` in this context to find the `XcodeBuildDestination`? My guess is actually yes but I wasn't sure.